### ISSUE TITLE
New Messages in slack not triggering the notification update in the app icon

### ISSUE THREAD

**Issue by tomasarg** [saenzramiro/rambox#1072]
When I get a new message I am not getting the Unread counter updated. It does not matter how I setup my notifications. Even with the most permissive ones I dont get anything 

	
	-
	> Rambox 0.5.10
	> Electron 1.7.4
	> darwin x64 16.7.0

**Comment by Thalagyrt** [comment]
Slack changed their message count CSS selectors.

For now, drop this in custom JS for each Slack you're in and it'll work:

```
function checkUnread() {
    var a = $(".p-channel_sidebar__channel--unread").length;
    var b = 0;
    $(".p-channel_sidebar__badge").each(function() {
        b += isNaN(parseInt($(this).html())) ? 0 : parseInt($(this).html())
    });
    updateBadge(a, b)
}

function updateBadge(a, b) {
    var c = b > 0 ? "(" + b + ") " : a > 0 ? "(•) " : "";
    document.title = c + originalTitle
}
var originalTitle = document.title;
setInterval(checkUnread, 3000);
```

I'm going to open a PR to fix this shortly.

**Comment by acollier88** [comment]
@Thalagyrt you're the best!

**Comment by Thalagyrt** [comment]
Happy to help out. :)

**Comment by mminks** [comment]
Thx.

**Comment by Feedy88** [comment]
Sorry if this is a stupid questions but where do I have to put the .js file? 😅

**Comment by JuDelCo** [comment]
Run it in the browser console (right click in the tab > toggle developer tools)

You must do this each time you start Rambox and per Slack tab (until a new rambox update arrives)

**Comment by diegok** [comment]
@JuDelCo no, there is a place in each service config for this. You place that JS snippet there, reload and it will be there until you manually remove it (after upgrading to the next version).

@Feedy88 Just go to the `Advanced` section down on the conf pane of each slack account and there is a box to paste that code.

**Comment by JuDelCo** [comment]
@diegok Oh, i never saw that option. Cool thanks !

**Comment by Ash258** [comment]
Fixed in #1076
